src/ol/events.js

```diff
@@ -48,11 +48,13 @@ export function listen(target, type, listener, thisArg, once) {
   if (once) {
     const originalListener = listener;
     /**
+     * @param {Event|import('./events/Event.js').default} event The event
+     * @return {void|boolean} When the function returns `false`, event propagation will stop.
```

Comment:
Is this documented elsewhere? I'm not sure we want this. We have a `stopPropagation()` function on the event, which - if I remember correctly - is the preferred way to stop propagation.

Reply:
That's documented here  and works with the listen function, just not with listenOnce.
https://github.com/openlayers/openlayers/blob/716c90a50335a825fb6813482123a45358f079e3/src/ol/events.js#L15-L21

Reply:
Maybe add a note about this change to the upgrade notes.